server/src/session.ts

```diff
@@ -1239,3 +1299,34 @@ function isTypeScriptFile(path: string): boolean {
 function isExternalTemplate(path: string): boolean {
   return !isTypeScriptFile(path);
 }
+
+interface CodeActionResolveData {
+  fixId?: string;
+  document: lsp.TextDocumentIdentifier;
+}
+
+/**
+ * Extract the fixAll action from `codeActions`
```

Comment:
Similar to my comment in the `onCodeActionResolve` function about describing why we only need to resolve actions for "fixAll" code actions, can you add a little bit more to this documentation as well? That is, what exactly we're extracting and mention that this aren't "complete" code actions, but will need to be made "complete" in the `onCodeActionResolve` function.